<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Header</class>
 <widget class="QWidget" name="Header">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>120</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(27, 26, 85);
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: #C8ACD6;
color:#C8ACD6 #17153B;
border-radius: 15px;
padding:10 15;
}</string>
     </property>
     <property name="text">
      <string>LogOut</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: #C8ACD6;
color: #17153B;
border-radius: 15px;
padding:10 15;
}</string>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #C8ACD6;</string>
     </property>
     <property name="text">
      <string>IQCS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: #C8ACD6;
color:#C8ACD6 #17153B;
border-radius: 15px;
padding:10 15;
}</string>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: #C8ACD6;
color: #17153B;
border-radius: 15px;
padding:10 15;
}</string>
     </property>
     <property name="text">
      <string>Report</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
